account-kit/react/package.json

```diff
@@ -46,10 +47,19 @@
     "test:run-e2e": "vitest run --config vitest.config.e2e.ts --passWithNoTests"
   },
   "devDependencies": {
+    "@storybook/addon-essentials": "^8.2.8",
```

Comment:
that's a lot of dependencies lol

Reply:
![](https://media2.giphy.com/media/v1.Y2lkPTc5MGI3NjExdDNlc2pxZ2QwbGMzY2F1MjBvYmV0Y3R6cXo1azJod201bHVzOXZhYiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/COYGe9rZvfiaQ/giphy.webp)